pytket/tests/qasm_test.py

```diff
@@ -49,7 +49,7 @@ def test_qasm_correct() -> None:
     assert c.depth() == 8
     coms = c.get_commands()
     assert len(coms) == 11
-    correct_str = "[Rz(1.5) q[3];, Rx(0.0375) q[3];, Rz(0.5) q[3];, CX q[0], q[3];, CZ q[0], q[1];, Rz(1.5) q[3];, Rx(1.9625) q[3];, CCX q[3], q[1], q[2];, Barrier q[0], q[3], q[2];, CU1(0.8) q[0], q[1];, U3(1, 0.5, 0.3) q[2];]"
+    correct_str = "[Rz(1.5) q[3];, Rx(0.0375) q[3];, XXPhase(0.0375) q[0], q[1];, Rz(0.5) q[3];, CX q[0], q[3];, CZ q[0], q[1];, Rz(1.5) q[3];, ZZPhase(0.0375) q[0], q[1];, Rx(1.9625) q[3];, CCX q[3], q[1], q[2];, Barrier q[0], q[3], q[2];, CU1(0.8) q[0], q[1];, U3(1, 0.5, 0.3) q[2];]"
```

Comment:
```suggestion
    correct_str = "[XXPhase(0.0375) q[0], q[1];, Rz(1.5) q[3];, ZZPhase(0.0375) q[0], q[1];, Rx(0.0375) q[3];, Rz(0.5) q[3];, CX q[0], q[3];, CZ q[0], q[1];, Rz(1.5) q[3];, Rx(1.9625) q[3];, CCX q[3], q[1], q[2];, Barrier q[0], q[3], q[2];, CU1(0.8) q[0], q[1];, U3(1, 0.5, 0.3) q[2];]"
```

Reply:
It should be like this because of the way TKET orders commands internally (though this isn't specified in the API, so it's not an ideal unit test.)

Reply:
Done.